views/index.ejs

```diff
@@ -7,6 +7,8 @@
     <meta name="zooniverse:deployed_commit" content="<%= process.env.HEAD_COMMIT %>">
     <meta name="zooniverse:build_env" content="<%= process.env.NODE_ENV %>">
     <meta name="zooniverse:build_date" content="<%= new Date() %>">
+    <link rel="icon" href="https://www.zooniverse.org/icon.svg">
```

Comment:
`sizes='any'` I don't think is required, but was included in [similar FEM changes](https://github.com/zooniverse/front-end-monorepo/pull/6310/files#diff-0954b021a4043c99d610e8caa0cc6f073a9f432ce3daabce7dc9d0dd8916a403R32).

Reply:
True! I'll add that here.